tokens/theme-semantics.json

```diff
@@ -42,6 +62,31 @@
           "description": "Text colour for external links."
         }
       }
+    },
+    "icon": {
+      "primary": {
+        "value": "{color.gray.1400}",
+        "type": "color"
+      },
+      "secondary": {
+        "value": "{color.gray.900}",
+        "type": "color"
+      },
+      "tertiary": {
+        "value": "{color.gray.800}",
+        "type": "color"
+      },
+      "quaternary": {
+        "value": "{color.gray.700}",
+        "type": "color",
+        "description": "⚠️ Lower than 3:1 contrast. Only use for non-essential icons. Do not rely exclusively on them."
+      },
+      "accent": {
+        "tertiary": {
```

Comment:
I'm not sure I understand this semantic? 
What constitutes a primary accent in the context of an icon?

Reply:
So, for icons, you have different color sets (default are gray, accent are green, etc.), and for each color set you have different degrees of emphasis (primary, secondary, etc.) so that we can control the contrast depending on the context and visual hierarchy.